### PR TITLE
[Documentation] The Contribution Guide update

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | yes |
| New docs?    | no |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

File: docs/contributing/support/support.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+At Sylius we have 3 main channels for communication and support.
+
+Slack
+-----
+
+Most of us use Slack for their online communication with co-workers. We know that it is not convenient to have another chat window open,
+therefore we've chosen Slack for the live communication with the community.
+
+We have setup following rooms so far:
```
Comment (NeverResponse) [inline docs/contributing/support/support.rst]: `set(space)up`

File: docs/contributing/support/support.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+#general - for discussions about Sylius development itself,
+#docs - for discussions related to the documentation,
+#support - for asking questions and helping others.
+If you have any suggestions, please let us know in the comments section below this post.
+
+Slack requires inviting new members, but this can be done automatically, just go to `slackinvite.me/to/sylius-devs <https://slackinvite.me/to/sylius-devs>`_,
+enter your e-mail and you will get an e-mail with invitation, where you can setup your login and password.
```
Comment (NeverResponse) [inline docs/contributing/support/support.rst]: I'm conflicted about using `e-mail`, it's got a kind of 90's feel to it. Would suggest changing to the dashless version, but i guess it's a case of taste. On the other hand, using email in whatever form twice in a span of 7 words is a no no.
`set(space)up` btw

File: docs/contributing/support/support.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+If possible, please use GitHub username - it will help us to recognize each other easily!
+
+StackOverflow
+-------------
+
+We encourage asking Sylius related questions on the `stackoverflow.com <http://stackoverflow.com>`_ platform.
+
+* Search for the question before asking, maybe someone did have this problem already,
```
Comment (NeverResponse) [inline docs/contributing/support/support.rst]: `maybe someone has already solved your problem` would be a better phrasing

File: docs/contributing/support/support.rst (hunk 1 above)
Comment (gabiudrescu) [inline docs/contributing/support/support.rst]: created / configured?

File: docs/contributing/support/support.rst [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+-----
+
+Most of us use Slack for their online communication with co-workers. We know that it is not convenient to have another chat window open,
+therefore we've chosen Slack for the live communication with the community.
+
+We have setup following rooms so far:
+
+#general - for discussions about Sylius development itself,
+#docs - for discussions related to the documentation,
+#support - for asking questions and helping others.
```
Comment (gabiudrescu) [inline docs/contributing/support/support.rst]: image

File: docs/contributing/support/support.rst (hunk 2 above)
Comment (gabiudrescu) [inline docs/contributing/support/support.rst]: How about: `enter your email and you will get an invitation`

**Comment by pjedrzejewski** [comment]
Great, thanks Magda! :)

**Pull request merged**